### PR TITLE
Application parts put together

### PR DESCRIPTION
**Pull request by HalloGene**
1. Added basic game loop
2. Added service for backend calls
3. Added statistics view

### CODE REVIEW

File: ui/grpc-web/etc/localhost.crt [hunk 1]
```diff
@@ -0,0 +1,18 @@
+-----BEGIN CERTIFICATE-----
```
Comment (mykyta-protsenko) [inline ui/grpc-web/etc/localhost.crt]: checking in certs and keys is a bad practice. It is compromised now. Not sure how it affects you but I my knee-jerk reaction is revoking the cert and adding it to .gitignore.

File: ui/grpc-web/.gitmodules [hunk 2]
```diff
@@ -0,0 +1,12 @@
+[submodule "third_party/grpc"]
```
Comment (mykyta-protsenko) [inline ui/grpc-web/.gitmodules]: can we avoid using .git submodules? They can complicate a build process.

File: ui/generated/gateway_pb.js [hunk 3]
```diff
@@ -0,0 +1,1617 @@
+/**
```
Comment (mykyta-protsenko) [inline ui/generated/gateway_pb.js]: Do we need to check in generated files?

File: ui/grpc-web/net/grpc/gateway/backend/backend.cc [hunk 4]
```diff
@@ -0,0 +1,29 @@
+/**
```
Comment (mykyta-protsenko) [inline ui/grpc-web/net/grpc/gateway/backend/backend.cc]: Not sure how C files are used in our project. Can you please clarify?

File: ui/grpc-web/etc/localhost.crt (hunk 1 above)
Comment (HalloGene) [reply ui/grpc-web/etc/localhost.crt]: this is publicly accessible certificate file from grpc-web project. i plan to remove grpc-web folder in the future.

File: ui/grpc-web/.gitmodules (hunk 2 above)
Comment (HalloGene) [reply ui/grpc-web/.gitmodules]: grpc-web folder will be removed

File: ui/generated/gateway_pb.js (hunk 3 above)
Comment (HalloGene) [reply ui/generated/gateway_pb.js]: yes for the time being. until the build process is configured.

File: ui/grpc-web/net/grpc/gateway/backend/backend.cc (hunk 4 above)
Comment (HalloGene) [reply ui/grpc-web/net/grpc/gateway/backend/backend.cc]: /grpc-web/ is our dependency. i checked in the entire project for the time being. i plan to remove this folder soon when build process is configured.

**Pull request merged**